docs/source/chaincode4ade.rst

```diff
@@ -383,7 +383,7 @@ Now let's compile your chaincode.
 
 .. code:: bash
 
-  go get -u github.com/hyperledger/fabric/core/chaincode/shim
+  go get -u github.com/hyperledger/fabric/core/chaincode/shim@v1.4
```

Comment:
@kemi04 @lindluni 
Did you guys try this?
I get "no matching versions for query "v1.4".

Reply:
I tried it, it worked for me, let me try again

Reply:
Ah, thats interesting, if you are in your GOPATH, this fails. If you are outside your GOPATH it succeeds by pulling the latest 1.4 release.

Reply:
After some playing around Dave and I discovered this command works but only if you are on the `release-1.4` branch with no `go.mod`